src/core/Form/Combobox/ComboboxItem/ComboboxItem.baseStyles.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-combobox-item {
+    &:focus {
+      outline: none;
+    }
+
+    & .fi-checkbox {
+      padding: 10px;
+      user-select: none;
+      ${font(theme)('actionElementInnerText')}
+
+      & mark {
```

Comment:
This might be a bit risky since we don't have any rests for mark-tag. Any global styles would be applied to these as well. Not sure if this is a problem, but maybe worth noting still.

Reply:
Not sure what change you are after, but do you mean that the `mark` "component" should also have some custom className?
e.g `fi-combobox-item_highlight-filter` or something?

Reply:
Sorry for not being descriptive with my comment. As this is wrapped inside .fi-checkbox it is probably fine the way it is now, but I'd still prefer the approach with classname rather than tag name.

Reply:
`mark` tag now has the `fi-combobox-item--query_highlight` class applied to it and it is used for the styling.